modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -266,6 +269,14 @@ public boolean isLocal(NetworkAddress node) {
         return clusterService.topologyService().localMember().address().equals(node);
     }
 
+    /** {@inheritDoc} */
+    public Map<IgniteUuid, List<byte[]>> lockedKeys(UUID txId) {
```

Comment:
I believe it'll be reworked after clarifying whole row place in lock methods.

Reply:
Seems that it will be fixed in the future and doesn't need to fix now.

Reply:
Yep, ok.